client/src/components/specification-table/styles.less

```diff
@@ -56,6 +56,10 @@
 .itm-rat {
     width: 150px;
 }
+// is it BEM? why do you use it only here then?
+// also it's wrong BEM
+// __ or -- or whatever people use to separate block&element
+// you can't have block__element__element. It breaks all idea of bem.
 .product-card__rating__stars {
```

Comment:
For rating, I get html content from the site ( $rating = $(".prod_header_reviews .rating")) , and in my component I just parse it as html, so the DOM is same as DOM on the site - > I didn't naming anything in this case, just add styling for it. :)